tests/github/export-diff.js

```diff
@@ -88,10 +97,26 @@ pullRequest.checkEnv()
       for (const changedPlugin of changedPlugins) {
         tasks = tasks.concat(usableTestFixtures.map(manFix => ({
           manFix,
-          pluginKey: changedPlugin
+          currentPluginKey: changedPlugin,
+          comparePluginKey: changedPlugin
         })));
       }
 
+      const addedPlugins = changedComponents.added.exports;
+      const removedPlugins = changedComponents.removed.exports;
+      for (const addedPlugin of addedPlugins) {
+        const previousPlugin = removedPlugins.find(
+          removedPlugin => pluginData.data[removedPlugin] && pluginData.data[removedPlugin].newPlugin === addedPlugin
```

Comment:
I think it does not need to be removed; think about when we added `qlcplus_4.12.0` alongside `qlcplus_4.11.2`; when we had set `previousVersion` then, it should have diffed the new plugin against the old one despite that not being removed.

Reply:
That diff would definitely be useful, but it's currently not even possible to use a still existing plugin as previous version of another plugin. That would require changes in make-plugin-data.js.

Reply:
How difficult would it be to implement this? I think that'd be quite useful.

Reply:
Implemented